src/components/Entry.vue

```diff
@@ -524,7 +560,7 @@ export default {
       await this.authenticateID(latestUserID);
       if (!this.isCurrentUserValid && this.validateCount == 0) {
         this.invalidLoginMessage =
-          "Student ID/Birthdate entered is incorrect. Please try again!<br/> Please register incase you are a new user.";
+          "Student ID entered is incorrect. Please try again!<br/> Please register incase you are a new user.";
```

Comment:
Registration form is there for these guys?

What about the contact number to call?